client/components/relatedProducts/Card.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { AiOutlineStar } from 'react-icons/ai';
+
+import '../../css/relatedProducts/Card.scss';
+
+const Card = ({ relatedProduct }) => (
```

Comment:
I would make each card a list item at the top level. This is best practices for accessibility. https://www.w3.org/WAI/tutorials/carousels/structure/

Reply:
Done.